### DISCUSSION TITLE
New command suggestion: playlistvar

### DISCUSSION THREAD

**Discussion by wolf109909** [ScureX/Takyon.PlayerVote#11] [Ideas]
So like, pandora made a pr about easier modification to each player's playlistvar like custom air accelerate and gravity and such, I think it's cool to let players decide what happens in the game or in next game. Like for example in public lobby rooms where ppl sometimes want to have 9kaa but sometimes they don't. If we have a vote command that can change how the game behaves in mid-game or in future games it would be easier for them to control the game on a voting basis.

**Comment by ScureX** [comment]
I kinda like that idea, however this would need voting in lobbies (which I'm not sure is possible, as i don't think i can display stuff). I may consider this in future versions where we have chat support etc.
